common/domain-setter.sub.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>A page that will likely be same-origin-domain but not same-origin</title>
+
+<script>
+"use strict";
+document.domain = "{{host}}";
```

Comment:
It can because it modifies the origin (not that I'd recommend using it for security). I guess the whole "relaxing" thing in the standard is a bit of misnomer, though it's what it's typically used for.